tests/model/test_model.py

```diff
@@ -1660,83 +1661,139 @@ def test_toggle_stream_pinned_status(self, mocker, model,
         (model.client.update_subscription_settings
          .assert_called_once_with(request))
 
-    @pytest.mark.parametrize('narrow, event, called', [
-        # Not in PM Narrow
-        ([], {}, False),
-        # Not in PM Narrow with sender
-        (
-            [['pm_with', 'iago@zulip.com']],
-            {
-                'type': 'typing',
-                'op': 'start',
-                'sender': {
-                    'user_id': 4,
-                    'email': 'hamlet@zulip.com'
-                },
-                'recipients': [{
-                    'user_id': 4,
-                    'email': 'hamlet@zulip.com'
-                }, {
-                    'user_id': 5,
-                    'email': 'iago@zulip.com'
-                }],
-                'id': 0
-            },
-            False,
-        ),
-        # In PM narrow with the sender, OP - 'start'
-        (
-            [['pm_with', 'hamlet@zulip.com']],
-            {
-                'op': 'start',
-                'sender': {
-                    'user_id': 4,
-                    'email': 'hamlet@zulip.com'
-                },
-                'recipients': [{
-                    'user_id': 4,
-                    'email': 'hamlet@zulip.com'
-                }, {
-                    'user_id': 5,
-                    'email': 'iago@zulip.com'
-                }],
-                'id': 0
-            },
-            True,
-        ),
-        # OP - 'stop'
-        (
-            [['pm_with', 'hamlet@zulip.com']],
-            {
-                'op': 'stop',
-                'sender': {
-                    'user_id': 4,
-                    'email': 'hamlet@zulip.com'
-                },
-                'recipients': [{
-                    'user_id': 4,
-                    'email': 'hamlet@zulip.com'
-                }, {
-                    'user_id': 5,
-                    'email': 'iago@zulip.com'
-                }],
-                'id': 0
-            },
-            True,
-        )
-    ], ids=['not_in_pm_narrow', 'not_in_pm_narrow_with_sender',
-            'start', 'stop'])
+    @pytest.mark.parametrize(
+        'narrow, event, animation_in_progress, expect_recipient_typing', [
```

Comment:
It should be interpreted as so:
```python
If "animation is in progress" and we "expect that the recipient is typing" then assert that this happens.
```
We don't expect animation status, maybe `is_animation_in_progress` is better?

Re `expected_recipient_typing_status` can be used if `recipient_typing_status` is the variable output of the function. Also from above discussion with Preet, 'status' implied more than 2 values and not a boolean.

I wanted to use `is_recipient_typing` but its the same name as boolean. How about `is_recipent_actually_typing`?
It would then be interpreted as:
```python
If "animation is in progress" and "recipient is actually typing" then assert that animation happens, etc...
```